hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/transaction/XATransactionCommitMessageTask.java

```diff
@@ -77,7 +77,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "transactionCommit";
```

Comment:
The method name for this is just commit. See XAResource.commit.

Reply:
changed.